bower.json

```diff
@@ -17,14 +17,14 @@
   "dependencies": {
```

Comment:
It's in bower. 
Basically npm stores dev dependencies and bower client dependencies. 

The tests uses customized BB and Mn builds.

For today standard is pretty convoluted but at the time the project started that was OK.

I changed the minimum to get working in decent shape in a modern env. Further improvements can be done later.